pss/pss.go

```diff
@@ -724,7 +725,7 @@ func sendMsg(p *Pss, sp *network.Peer, msg *message.Message) bool {
 		metrics.GetOrRegisterCounter("pss.pp.send.error", nil).Inc(1)
 		log.Error(err.Error())
 	}
-
+	metrics.GetOrRegisterCounter("pss.pp.send", nil).Inc(1)
```

Comment:
I think a timer for the whole function would provide more visibility, than just a counter after the pp.Send has returned.